src/translations/components/TranslationsProductsPage/TranslationsProductsPage.tsx

```diff
@@ -123,6 +125,28 @@ const TranslationsProductsPage: React.FC<TranslationsProductsPageProps> = ({
         onDiscard={onDiscard}
         onSubmit={onSubmit}
       />
+      <CardSpacer />
+      <TranslationFields
+        activeField={activeField}
+        disabled={disabled}
+        initialState={true}
+        title={intl.formatMessage(commonMessages.translationAttributes)}
+        fields={
+          data?.attributeValues?.map(attrVal => ({
```

Comment:
since this is used a couple of times should be a util function

Reply:
This one in particular is used twice, I'd leave it as is right now.